### PR TITLE
update(PopoutPlayer): Close popout when click on close button

### LINKED ISSUE #58: Clicking "close" in pop out player will not close it
**Issue:** When you have the pop out player open in a call, clicking the close button seems to just force the pop out player into fullscreen instead of exiting.

**Screen recording:**

https://user-images.githubusercontent.com/93608357/210623478-f80431ef-0c86-447a-aa30-8331fa727b5a.mov

### PR DESCRIPTION
**Pull request by lgmarchi**
### 1. Close pop out player with close button

https://user-images.githubusercontent.com/63157656/211160679-2d6ecc91-4df4-4a48-a9dc-1a0a8a2f307a.mov

### 2. Close pop out player with popout button 

https://user-images.githubusercontent.com/63157656/211370710-521f5585-63bf-43f4-a99f-d9b7bc2a355b.mov

### 3. Close pop out player with end call buttons 

https://user-images.githubusercontent.com/63157656/211370923-7e3c9e09-093a-4a9d-a13b-a204ae462588.mov

### 4. Close Pop out player if we start a call with other user 

https://user-images.githubusercontent.com/63157656/211371137-f91d4559-fca7-4c15-9ec9-18462a74ae2e.mov

### Which issue(s) this PR fixes 🔨

- Resolve #58

### CODE REVIEW

File: icons/src/outline.rs [hunk 1]
```diff
@@ -104,6 +104,8 @@ pub enum Shape {
     ChatBubbleLeft,
     ChatBubbleOvalLeftEllipsis,
     ChatBubbleOvalLeft,
+    ChatPlus,
```
Comment (sdwoodbury) [inline icons/src/outline.rs:107]: I believe the changes to this file are from `dev`. I pulled `dev` and built it and got what look like the same changes as here. so I think it's OK to have the changes to icons here.

File: ui/src/state/action.rs [hunk 2]
```diff
@@ -41,8 +42,10 @@ pub enum Action {
     /// adds an overlay. currently only used for demonstration purposes
     AddOverlay(Weak<WebView>),
     /// used for the popout player or media player
-    SetPopout(Rc<WebView>),
-    ClearPopout,
+    /// use state.ui.clear_popout to close the popout. this function
```
Comment (sdwoodbury) [inline ui/src/state/action.rs]: these comments are no longer valid

File: ui/src/state/ui.rs [hunk 3]
```diff
@@ -98,30 +102,25 @@ pub struct Call {
...
-        if let Some(view) = self.popout_view.as_ref() {
-            view.evaluate_script("close()")
-                .expect("failed to close webview");
-        };
-    }
-}
-
 impl Call {
-    pub fn new(popout_view: Option<Rc<WebView>>) -> Self {
+    pub fn new(popout_view: Option<WindowId>) -> Self {
```
Comment (sdwoodbury) [inline ui/src/state/ui.rs]: i should have changed the name of `popout_view` to `popout_window_id`

**Review by sdwoodbury — APPROVED** [review]

**Comment by phillsatellite** [comment]
Only found 2 minor things when testing everything else was working great!

1. If User closes popout player by clicking "Close" within the popout player when they try to open pop out again it will not open

https://user-images.githubusercontent.com/93608357/211383607-aed708e4-f721-4782-86ff-237f798ad0d9.mov

2. Second issue I had was I couldnt get the Popout to close with the Popout button. I had tripple checked that I had the latest commits and did cargo update and clean but still couldnt get that feature working

https://user-images.githubusercontent.com/93608357/211384609-4f2965fe-5757-4ef6-9dd4-255dee7770d5.mov

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**